client/cmdmain.c

```diff
@@ -33,6 +34,7 @@ static command_t CommandTable[] = {
 	{"hf",		CmdHF,		1, "{ High Frequency commands... }"},
 	{"hw",		CmdHW,		1, "{ Hardware commands... }"},
 	{"lf",		CmdLF,		1, "{ Low Frequency commands... }"},
+	{"rem",		CmdRem, 	1, "{ Display remark }"},
```

Comment:
may a different text...   "Add text to row in log file" ?

Reply:
ok

Reply:
Done: new commit